src/deriver/consumer.py

```
@@ -133,5 +151,76 @@ async def process_representation_batch(
+
+        elif deletion_type == "observation":
+            try:
+                await crud.delete_document_by_id(
+                    db, workspace_name=workspace_name, document_id=resource_id
+                )
+                logger.info(
+                    "Successfully deleted observation %s in workspace %s",
+                    resource_id,
+                    workspace_name,
+                )
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)